configs/pushgateway/README.md

```diff
@@ -0,0 +1,13 @@
+# Prometheus pushgateway
+
+## Install
+
+```
+helm install pushgateway --namespace monitoring .
+```
+
+## Uninstall
+
+```
+helm install pushgateway --namespace monitoring
+```
```

Comment:
Why do we need pushgateway configs here? They are already available here: https://github.com/kinvolk/service-mesh-benchmark/tree/master/configs/pushgateway.

Reply:
I thought we could have it in a same repository? If you want I can remove it

Reply:
It should ideally be in only one repository. I don't have an opinion on which repository is best for it.

Reply:
I can remove it then and point it to service mesh repo